### DISCUSSION TITLE
Support all native data types, functions and operators in shapes' WHERE clauses when PGLite is used on the client

### DISCUSSION THREAD

**Discussion by alco** [electric-sql/electric#1306] [Feature requests]
This should be possible because there's no need for mapping between PGLite, at least on a conceptual level.

The only things to pay special attention to are differences between Postgres versions and the use of different extensions on the server and on the client.

**Comment by KyleAMathews** [comment]
👋 we've been working the last month on a rebuild of the Electric server over at a temporary repo https://github.com/electric-sql/electric-next/

You can read more about why we made the decision at https://next.electric-sql.com/about

We're really excited about all the new possibilities the new server brings and we hope you'll check it out soon and give us your feedback.

We're now moving the temporary repo back here. As part of that migration we're closing all the old issues, PRs, and discussions. We really appreciate you taking the time to start this discussion! If it's still relevant in the new system, please reopen.
